dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_postgresql.sql

```diff
@@ -1953,11 +1953,11 @@ CREATE TABLE t_ds_cluster(
 );
 
 --
--- Table structure for t_ds_fav
+-- Table structure for t_ds_fav_task
 --
 
-DROP TABLE IF EXISTS t_ds_fav;
-CREATE TABLE t_ds_fav
+DROP TABLE IF EXISTS t_ds_fav_task;
+CREATE TABLE t_ds_fav_task
```

Comment:
should we also change mysql DDL?

Reply:
oh I got it, in `dolphinscheduler_mysql.sql` we already with the name `t_ds_fav_task `

Reply:
MySQL DDL already done before, h2 and postgre are forgot

Reply:
Thanks for your clarification